spec/models/post_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'A beautiful beach',
+              post_counter: 0)
+  subject { Post.new(user_id: user.id, title: 'Hello', text: 'This is my first post') }
```

Comment:
- [ ] The same thing with line 6 in the post_spec file. Try to define the `user` used for post creation.